fendermint/app/src/options/rpc.rs

```diff
@@ -145,8 +147,8 @@ pub struct TransArgs {
     /// Name of chain the for which the message will be signed.
     #[arg(long, short, env = "FM_CHAIN_NAME")]
     pub chain_name: String,
-    /// Amount of tokens to send, in atto.
-    #[arg(long, short, value_parser = parse_token_amount, default_value = "0")]
+    /// Amount of tokens to send, in full FIL, not atto.
```

Comment:
When I read "full FIL" I imagine it will be an _integer_ that contains whole FIL amounts, 1, 2, 3, etc. Maybe "fractional FIL"? But if "full FIL" is the adopted nomenclature it's okay, now I know.

Reply:
I usually call `full_fil` because it is not subdivided into `atto` or other subunits, but weak opinion weakly held. If you think it is clearer as `fractional_FIL` I am happy to change it.